### ISSUE TITLE
Windows 7

### ISSUE THREAD

**Issue by harryray33** [Genymobile/scrcpy#5043]
Does this work with Win 7?

Thanks

**Comment by rom1v** [comment]
Yes, but you might need an older version of adb: https://github.com/Genymobile/scrcpy/issues/4391

**Comment by bt4ibwem8** [comment]
for me it works fine with the latest v35 version of adb on win7
